browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -359,6 +361,171 @@ test.describe('view program statuses', () => {
 
       expect(await adminPrograms.getNoteContent()).toBe(noteText)
     })
+
+    test.describe('email status updates work correctly with PAI flag on', () => {
+      test.beforeAll(async () => {
+        const {page, adminPrograms, adminQuestions} = ctx
+        await enableFeatureFlag(
+          page,
+          'primary_applicant_info_questions_enabled',
+        )
+        await loginAsAdmin(page)
+        await adminQuestions.addEmailQuestion({
+          questionName: 'Email',
+          universal: true,
+          primaryApplicantInfo: true,
+        })
+        await adminPrograms.editProgram(programWithStatusesName)
+        await adminPrograms.editProgramBlock(
+          programWithStatusesName,
+          'block description',
+          ['Email'],
+        )
+        await adminPrograms.publishAllDrafts()
+        await logout(page)
+      })
+      test.beforeEach(async () => {
+        const {page} = ctx
+        // Logout before each test so we can start from the homescreen as an applicant
+        await logout(page)
+      })
+      test.afterAll(async () => {
+        const {page} = ctx
+        await disableFeatureFlag(
+          page,
+          'primary_applicant_info_questions_enabled',
+        )
+      })
+      test('email is displayed and sent for guest user that has answered the PAI email question', async () => {
+        const {page, applicantQuestions, adminPrograms} = ctx
+        const guestEmail = 'guestemail@example.com'
+
+        await applicantQuestions.applyProgram(programWithStatusesName)
+        await applicantQuestions.answerEmailQuestion(guestEmail)
+        await applicantQuestions.clickNext()
+        await applicantQuestions.submitFromReviewPage()
+        await logout(page)
+
+        await loginAsProgramAdmin(page)
+        await adminPrograms.viewApplications(programWithStatusesName)
+        await adminPrograms.viewApplicationForApplicant('Guest')
+
+        const emailsBefore = supportsEmailInspection()
+          ? await extractEmailsForRecipient(page, guestEmail)
+          : []
+
+        const modal =
+          await adminPrograms.setStatusOptionAndAwaitModal(emailStatusName)
+
+        const notifyCheckbox = await modal.$('input[type=checkbox]')
```

Comment:
`$` is deprecated. You should switch to a [locator](https://playwright.dev/docs/locators), preferable using one of the `page.getByXxx` methods and only fall back to `page.locator` as a last resort.

Reply:
Fun fact: one reason locators are better is because if they find multiple match they error. Older ways will go with the first one found :|

Reply:
I couldn't get the locator working so I left this with `$` for now.